src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -578,7 +578,16 @@
 examples:
   - name: Add bg extension.
```

Comment:
can you change "bg" for custom scrip vm extension?

Reply:
updated